README.md

```diff
@@ -24,6 +24,5 @@ This is an addon with some utils and helpers to get country names and calling co
 
 ## Contributing
 
-Any contribution is welcome. Please read our [guidelines](CONTRIBUTING.md).
```

Comment:
Why?

Reply:
Probably removed during the `ember-cli-update`. Reverted.